salt/states/zpool.py

```diff
@@ -30,6 +30,23 @@
               /dev/disk2
               /dev/disk3
 
+    partitionpool:
+      zpool.present:
+        - config:
+            import: false
+            force: true
+        - properties:
+            comment: disk partition salty storage pool
+            ashift: '12'
+            feature@lz4_compress: enabled
+        - filesystem_properties:
+            compression: lz4
+            atime: on
+            relatime: on
+        - layout:
+            disk-0:
```

Comment:
Just managed to do have a quick look in the minion's log and I believe the error was:

```
2016-09-08 14:08:22,351 [salt.loaded.int.module.cmdmod][ERROR   ][12224] Command '/sbin/zpool create -O mountpoint=/media/zfsroot zfsroot /dev/disk/by-uuid/2b34a3b2-f490-4367-a71f-053a18bc9720' failed with return 
code: 1
2016-09-08 14:08:22,352 [salt.loaded.int.module.cmdmod][ERROR   ][12224] stderr: invalid vdev specification
use '-f' to override the following errors:
/dev/disk/by-uuid/2b34a3b2-f490-4367-a71f-053a18bc9720 contains a filesystem of type 'swap'
```

So it appears that I wasn't using `force: true` at the time.  Could that be the reason for the `invalid vdev specification` error?